packages/application-generic/src/factories/mail/handlers/resend.handler.ts

```diff
@@ -6,10 +6,11 @@ export class ResendHandler extends BaseHandler {
   constructor() {
     super('resend', ChannelTypeEnum.EMAIL);
   }
-  buildProvider(credentials: ICredentials, from?: string) {
-    const config: { apiKey: string; from: string } = {
+  buildProvider(credentials: ICredentials, from?: string, senderName?: string) {
```

Comment:
`credentials: ICredentials` already includes the senderName.
So we don't need to pass senderName as an extra param. Also in the current implementation. In the `mail.factory.ts` file, only 2 params are being passed for all the providers.